sound/hybrid_font.h

```diff
@@ -469,7 +469,10 @@ class HybridFont : public SaberBase, public Looper {
             hum_player_->set_fade_time(0.2);
             hum_player_->FadeAndStop();
             hum_player_.Free();
-          }
+          } else {
+          state_ = STATE_HUM_FADE_OUT;
```

Comment:
Fix looks reasonable, but indentation looks bad.
Could be github messing up, please check.

Reply:
Indeed. It looks ok online, but it's weird when pulled down.
All the adds you did look off too. Weird. Fixed.
![Screen Shot 2022-01-10 at 8 57 44 PM](https://user-images.githubusercontent.com/53964195/148868036-7309d01b-195c-4535-bbb3-ee8b70e688b4.jpg)